pkg/doc/presexch/models.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package presexch
+
+import (
+	"github.com/hyperledger/aries-framework-go/component/models/presexch"
+)
+
+// PresentationSubmission is the container for the descriptor_map:
```

Comment:
Why did you redefine all these types?

Reply:
They're not redefined - this package is the original location of all these types, so it's the import that all consumers of the package are currently using. Rather than break all consumers of refactored packages, I've been doing this sort of thing in all my refactor PRs. Original names in original locations can continue to be used, and migrated to using the new packages/modules at our convenience.

Reply:
Makes sense, thanks!